sdk/storage/Azure.Storage.Blobs/src/BlobLeaseClient.cs

```diff
@@ -608,7 +608,9 @@ private void EnsureClient()
         /// </remarks>
         public virtual async Task<Response<ReleasedObjectInfo>> ReleaseInternal(
             RequestConditions conditions,
+#pragma warning disable AZC0105
```

Comment:
`BlobLeaseClient.ReleaseInternal` is a public method of a public type. @tg-msft, have we published this method?

Reply:
Yes - it was an unfortunate mistake.

Reply:
Do we have to keep it that way?

Reply:
Yes. We cannot remove a method once it shipped in a stable library.